packages/website/app/components/headerNav.tsx

```diff
@@ -1,33 +1,43 @@
-import { Link, useMatches } from '@remix-run/react'
-
-const HeaderNav = () => {
-	const matches = useMatches()
-	return (
-		<div w-justify="between" w-item="center" w-flex="~" w-p="y-12 x-10" w-pos="sticky" w-top="0">
-			<ul w-flex="~" w-children:p="r-4">
-				{matches.slice(0, -1).map(match => <li
-					key={match.id.toString()}
-					data-sep="/ "
-					className="not-last:after:content-raw-[attr(data-sep)]"
-				>{match.handle?.breadcrumb(match)}</li>
-				)}
-			</ul>
-			<ul w-flex="~" w-children:p="l-10">
-				<li>
-					<button>
-						<div className="i-ant-design:search-outlined" w-display="inline-block"/>Search
-					</button>
-				</li>
-				<li>
-					<button>
-						<Link to="/login" w-decoration="none">
-							<div className="i-ant-design:user-outlined" w-display="inline-block"/>Login
-						</Link>
-					</button>
-				</li>
-			</ul>
-		</div>
-	)
-}
-
-export default HeaderNav
+import { Link, useMatches } from '@remix-run/react'
+import uuid from 'react-uuid'
+
+
+const HeaderNav = () => {
+	const matches = useMatches()
+	return (
+		<div className="headernav top-0 sticky justify-between flex bg-white items-center py-3 px-2"
+			w-p="12px 10px"
+			w-bg='white'  
+			w-w='100vw'
+			w-h='50px'
+			w-flex='row'
+			w-top='0'
+			w-items='center'>
+			<div className="left flex">
+				{matches
+					.filter((match) => match.handle && match.handle.breadcrumb)
+					.map((match, index) => (
+						<div className="flex" key={uuid()}>

```

Comment:
```suggestion
						<div className="flex" key={match.id}>

```

That is enough, no need for `react-uuid`.